public/app/features/explore/ExplorePaneContainer.tsx

```diff
@@ -87,7 +101,8 @@ class ExplorePaneContainerUnconnected extends React.PureComponent<Props> {
   };
 
   render() {
-    const exploreClass = this.props.split ? 'explore explore-split' : 'explore';
+    const styles = getStyles();
+    const exploreClass = this.props.split ? cx(styles.explore, styles.exploreSplit) : cx(styles.explore);
```

Comment:
`cx` [handles conditional styles](https://emotion.sh/docs/@emotion/css#cx), you can use one of these for better readability:
```suggestion
    const exploreClass = cx(styles.explore, this.props.split && styles.exploreSplit);
```

or 
```suggestion
    const exploreClass = cx(styles.explore, { [styles.exploreSplit]: this.props.split });
```

(i think the first reads better in this case)